Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -599,12 +609,10 @@ public final class _ExecutionContext {
 
                 stringAddrs.withUnsafeMutableBufferPointer { stringAddrsBuffer in
                     var cArgs = [stringAddrsBuffer.baseAddress.map(UnsafeMutablePointer.init)]
```

Comment:
This can be simplified in the same way: Instead of creating a zero-element array, create a `var` and pass it as an inout reference.

Reply:
Good point. Done in the last commit.